src/sentry/static/sentry/app/views/settings/components/settingsPageHeader.jsx

```diff
@@ -4,20 +4,33 @@ import styled from 'react-emotion';
 import {Flex, Box} from 'grid-emotion';
 import space from 'app/styles/space';
 
+const Heading = styled(Flex)`
+  margin: ${space(3)} 0;
+  flex: 1;
+`;
+
+const Title = styled(Heading)`
+  font-size: 20px;
+  font-weight: bold;
+`;
+
 class SettingsPageHeading extends React.Component {
   static propTypes = {
     icon: PropTypes.node,
     title: PropTypes.node,
     action: PropTypes.node,
     tabs: PropTypes.node,
+    noStyles: PropTypes.bool,
   };
 
   render() {
+    const StyledTitle = this.props.noStyles ? Heading : Title;
```

Comment:
nit/imo: `noStyles` is presentational and would be better off being handled by a single styled component instead of this component.

Reply:
Yeah that's a good point. I just like that this keeps actions and tabs

Reply:
actions/tabs wouldn't need to change?

i.e. if you had the old `Title` styled component and did something like:

```
${p => !p.noStyles && `
  font-size: 20px;
  font-weight: bold;
`};
```

Reply:
Ahh! I see what you're saying!

I interpreted this as you saying just to make a new one-off styled component for the title. I didn't realize I could do what you have there. Will do that, looks great.